src/Schedule.h

```diff
@@ -232,10 +241,21 @@ class Schedule {
      * true. See \ref Func::specialize */
     // @{
     const std::vector<Specialization> &specializations() const;
+    std::vector<Specialization> &specializations();
```

Comment:
Is mutable access to the specializations still necessary, given that the deep copy is a method?